mypy.ini

```diff
@@ -28,7 +28,7 @@ ignore_missing_imports = True
 [mypy-lxml.*]
 ignore_missing_imports = True
 
-[mypy-marshmallow.*]
+[mypy-OpenSSL.*]
 ignore_missing_imports = True
```

Comment:
**It's OK** to remove `mypy-marshmallow`, but replacing it with `mypy-OpenSSL` is a mistake.

```suggestion
```